### DISCUSSION TITLE
How to run qutebrowser on wayland natively (no XWayland)

### DISCUSSION THREAD

**Discussion by fritzrehde** [qutebrowser/qutebrowser#7654] [Q&A]
I have installed `qt5-wayland` and `qt6-wayland`, but qutebrowser still launches as an XWayland app. Do I have to change some other settings as well?

**Comment by The-Compiler** [comment]
Set either `QT_QPA_PLATFORM=wayland` in your environment, or set `qt.force_platform` to `wayland` - for the former, see the [Arch Wiki on how to set up Wayland](https://wiki.archlinux.org/title/wayland#Qt).
